storagenode/nodestats/service.go

```diff
@@ -145,7 +232,9 @@ func (s *Service) GetDailyStorageUsedForSatellite(ctx context.Context, satellite
 }
 
 // DialNodeStats dials GRPC NodeStats client for the satellite by id
-func (s *Service) DialNodeStats(ctx context.Context, satelliteID storj.NodeID) (*Client, error) {
+func (s *Service) DialNodeStats(ctx context.Context, satelliteID storj.NodeID) (_ *Client, err error) {
+	defer mon.Task()(&ctx)(&err)
+
 	satellite, err := s.kademlia.FindNode(ctx, satelliteID)
```

Comment:
We should have the necessary information in trust pool, we shouldn't search for satellites on the kademlia.